### PR TITLE
[AAE-10697] Angular 14 upgrade

### PR DESCRIPTION
**Pull request by DenysVuika**
Upgrade templates to Angular 14
Setup GitHub Actions for the CI/CD

image

https://alfresco.atlassian.net/browse/AAE-10697

### CODE REVIEW

File: app/templates/adf-cli-activiti-community-template/src/app/stencils.module.ts [hunk 1]
```diff
@@ -29,16 +29,12 @@ export class CustomStencil01 extends WidgetComponent {
 
 @NgModule({
     declarations: [
-      CustomEditorComponent,
-      CustomStencil01
+        CustomEditorComponent,
+        CustomStencil01
```
Comment (nikita-web-ua) [inline app/templates/adf-cli-activiti-community-template/src/app/stencils.module.ts:33]: Are those extra spaces made on purpose?

File: app/templates/adf-cli-activiti-community-acs-template/src/app/stencils.module.ts [hunk 2]
```diff
@@ -29,16 +29,12 @@ export class CustomStencil01 extends WidgetComponent {
 
 @NgModule({
     declarations: [
-      CustomEditorComponent,
-      CustomStencil01
+        CustomEditorComponent,
+        CustomStencil01
```
Comment (nikita-web-ua) [inline app/templates/adf-cli-activiti-community-acs-template/src/app/stencils.module.ts:33]: same gaps?

File: app/templates/adf-cli-apa-acs-template/src/app/stencils.module.ts [hunk 3]
```diff
@@ -1,44 +1,26 @@
...
-    }
 }
 
 @Component({
-    selector: 'custom-stencil-01',
-    template: `<div style="color: green">ADF version of custom Activiti stencil</div>`
+  selector: "custom-stencil-01",
+  template: `<div style="color: green">
+    ADF version of custom Activiti stencil
+  </div>`,
```
Comment (nikita-web-ua) [inline app/templates/adf-cli-apa-acs-template/src/app/stencils.module.ts:17]: trailing commas intentionally?

File: app/templates/adf-cli-activiti-community-acs-template/src/app/stencils.module.ts (hunk 2 above)
Comment (DenysVuika) [reply app/templates/adf-cli-activiti-community-acs-template/src/app/stencils.module.ts]: fixed

File: app/templates/adf-cli-apa-acs-template/src/app/stencils.module.ts (hunk 3 above)
Comment (DenysVuika) [reply app/templates/adf-cli-apa-acs-template/src/app/stencils.module.ts]: those were part of the old linter, should be ok but I've cleaned anyway, thanks

**Pull request merged**